meshtastic/__init__.py

```diff
@@ -227,8 +229,11 @@ def getTimeAgo(ts):
         for i, row in enumerate(rows):
             row['N'] = i+1
 
-        print(tabulate(rows, headers='keys', missingval='N/A',
-                       tablefmt='fancy_grid'), file=file)
```

Comment:
maybe someone should check for dependencies to the file=file property, i removed it on this

Reply:
Thanks for the PR.  I don't quite follow this part though, is there a reason you removed the option of emitting the showNodes() output to a destination other than stdout)?

Reply:
As i changed the prints to assignments, i felt this "file" output was not necessary on this methods, so i tested without them and it works.

Reply:
ok - sounds good!